crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations.rs

```diff
@@ -602,10 +603,24 @@ impl ProtocolCryptographicData {
                         malicious_parties,
                         private_output,
                     } => {
-                        let signature: ECDSASecp256k1Signature =
-                            bcs::from_bytes(&public_output_value)?;
+                        let public_output_value = match data.curve {
+                            DWalletMPCNetworkKeyScheme::Secp256k1 => {
+                                let signature: ECDSASecp256k1Signature =
```

Comment:
Either add support for TaprootSignature if its schnorr, or link it to an issue to add support for schnorr